<ui version="4.0" >
 <class>PartitionManagerKCMBase</class>
 <widget class="QWidget" name="PartitionManagerKCMBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>684</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item rowspan="8" row="0" column="0">
    <widget class="QSplitter" name="m_SplitterHorizontal" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="m_SplitterVertical" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="ListDevices" native="1" name="m_ListDevices" >
       <property name="minimumSize" >
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="ListOperations" native="1" name="m_ListOperations" />
     </widget>
     <widget class="QWidget">
      <layout class="QVBoxLayout">
       <item>
        <widget class="KToolBar" native="1" name="m_ToolBar" />
       </item>
       <item>
        <widget class="PartitionManagerWidget" native="1" name="m_PartitionManagerWidget" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="newButton">
        <property name="text" >
         <string>New</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="resizeButton">
        <property name="text" >
         <string>Resize/Move</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="deleteButton">
        <property name="text" >
         <string>Delete</string>
        </property>
       </widget>
      </item>
   <item row="3" column="1" >
    <widget class="QPushButton" name="shredButton" >
     <property name="text" >
      <string>Shred</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QPushButton" name="unmountButton" >
     <property name="text" >
      <string>Unmount</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PartitionManagerWidget</class>
   <extends>QWidget</extends>
   <header>gui/partitionmanagerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ListDevices</class>
   <extends>QWidget</extends>
   <header>gui/listdevices.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ListOperations</class>
   <extends>QWidget</extends>
   <header>gui/listoperations.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KToolBar</class>
   <extends>QWidget</extends>
   <header location="global" >ktoolbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
